docs/github-actions.rst

```diff
@@ -36,4 +36,24 @@ Most commonly, you can use it to validate an instance (some data) using a specif
 replacing ``lua-jsonschema`` and the filenames with your implementation and schema of choice.
 For full details on the commands available, see the `CLI documentation <cli>`.
 
+Including Bowtie Output in Workflow summary
+-------------------------------------------
+
+Commands such as bowtie summary, bowtie smoke and bowtie info support outputting in markdown format using the
+--format markdown flag. This output can be shown in the workflow summary for eg. output validation results for
```

Comment:
Would also be good to link to the GitHub Actions docs for workflow summary / to explain what GITHUB_STEP_SUMMARY is.